decidim-core/app/models/decidim/pad.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module Decidim
+  # This class allows you to interact with pad stored in an Etherpad Lite server.
+  class Pad
+    def initialize(pad_id)
+      @id = pad_id
+      @api_key = Decidim.etherpad.fetch(:api_key)
+      @api_version = Decidim.etherpad.fetch(:api_version, "1.2.1")
+      @uri = URI.parse(Decidim.etherpad.fetch(:server))
+    end
+
+    attr_reader :id
+
+    # Read only means that pad is not writable.
+    def read_only_id
+      @read_only_id ||= resolve(:getReadOnlyID, { padID: id })[:readOnlyID]
```

Comment:
Can we add a spec that tests this API request against a "dummy" Etherpad API (stub request) which responds with the expected Etherpad API response (check from the Etherpad docs)?

Check e.g. [`here_spec.rb`](https://github.com/decidim/decidim/blob/9f8e747f8db7e0aeb8618494325555600f87998c/decidim-core/spec/lib/map/provider/geocoding/here_spec.rb) to take inspiration from.